### ISSUE TITLE
The title level of org-mode and markdown are not clear

### ISSUE THREAD

**Issue by re-mx** [bbatsov/zenburn-emacs#308]
Hello, and forgive me for my poor English.

I really like your theme, but I found some small problems that bothered me.

In other themes, the org-mode and markdown levels are easy to distinguish. But in your theme, there seems to be no way to distinguish anything other than color. This is very difficult for me with color blindness.

I hope to improve, such as the difference in font size.
But I am an elisp rookie, so I can't provide PR for this, sorry.

Thank you.

Some screenshots:
<img width="1280" alt="2" src="https://user-images.githubusercontent.com/26828933/42731034-7a1b519c-8837-11e8-97ca-d5fd192a1f81.png">

<img width="1280" alt="1" src="https://user-images.githubusercontent.com/26828933/42731037-83046e6a-8837-11e8-8aae-cf4b97238d0c.png">

Leuven Theme:
<img width="1280" alt="3" src="https://user-images.githubusercontent.com/26828933/42731045-a7d81174-8837-11e8-9a30-3817db23eabf.png">

**Comment by re-mx** [comment]
thanks for your reply @georgek 

If this is the design of zenburn, then I might really need to maintain a version of myself.

I may not close this issue because I feel that my question has not been answered.
But if this issue is stupid, please close it. 

Thank you.

**Comment by ksjogo** [comment]
You are able to just overwrite single faces inside your emacs config, so you wouldn't have to keep the whole version along and can just use upstream.

**Comment by georgek** [comment]
Yes. I use to do that actually. I just had a separate mods file which I would load after the theme: https://github.com/georgek/dot-emacs/blob/master/themes/zenburn-mods.el

The reason I inlined mine was because I was having strange problems when byte-compiling emacs lisp. For some reason elements of the style outside the one loaded by `deftheme` get lost.

**Comment by bbatsov** [comment]
For what is worth - I'm fine with adding a configuration option (defcustom) about using dynamic-font-sizes for headings/titles/etc. I generally dislike them as I think the markup itself gives enough info about the heading size and I like uniform fonts in source buffers, but I know many people have different opinion, so I'm fine with adding this if it's something opt-in.

**Comment by georgek** [comment]
I'll have a go at adding this feature over the weekend then, unless someone else gets to it first.
